web/src/tippyjs.ts

```diff
@@ -601,6 +601,12 @@ export function initialize(): void {
         },
     });
 
+    tippy.delegate("body", {
+        target: ".personal-menu-clear-status",
```

Comment:
It's a bit confusing to have this selector name be used for two different components; probably ideally we'd just use a different CSS class name for the one in the user card, and list them both here.